R/FilteredData.R

```diff
@@ -247,19 +247,14 @@ FilteredData <- R6::R6Class( # nolint
     #' @param dataset_2 (`character(1)`) other dataset name
     #' @return (`named character`) vector with column names
     get_join_keys = function(dataset_1, dataset_2) {
```

Comment:
So this keeps the previous behaviour - I'm not sure that is what we want though?

Reply:
To be done here https://github.com/insightsengineering/teal.data/issues/78